js/components/addressBook/addressBook_controller.js

```diff
@@ -141,6 +141,12 @@ angular.module('contactsApp')
 	};
 
 	ctrl.onSelectSharee = function (item) {
+		// Prevent settings to slide down
+		$('#app-settings-header > button').data('apps-slide-toggle', false);
```

Comment:
This is an ugly trick. But core is broken for popups in the settings area.
Here is the issue: https://github.com/nextcloud/server/blob/25acce7e0d992b10b930472b8138240b7bd9af13/core/js/apps.js#L108